yarn.lock

```diff
@@ -20260,9 +20334,9 @@ zen-observable@0.8.15:
   resolved "https://registry.yarnpkg.com/zen-observable/-/zen-observable-0.8.15.tgz#96415c512d8e3ffd920afd3889604e30b9eaac15"
   integrity sha512-PQ2PC7R9rslx84ndNBZB/Dkv8V8fZEpk83RLgXtYd0fwUgEjseMn1Dgajh2x6S8QbZAFa9p2qVCEuYZNgve0dQ==
 
-zustand@^4.0.0-rc.0, zustand@^4.0.0-rc.1:
-  version "4.0.0"
-  resolved "https://registry.yarnpkg.com/zustand/-/zustand-4.0.0.tgz#739cba69209ffe67b31e7d6741c25b51496114a7"
-  integrity sha512-OrsfQTnRXF1LZ9/vR/IqN9ws5EXUhb149xmPjErZnUrkgxS/gAHGy2dPNIVkVvoxrVe1sIydn4JjF0dYHmGeeQ==
+zustand@^4.0.0-rc.0, zustand@^4.3.5, zustand@^4.3.6:
```

Comment:
Do we know where's `zustand@^4.0.0-rc.0` coming from? I don't see it being mentioned anywhere in the code. I wonder if removing it and running `yarn` again would update the lock file. Looks like a leftover from previous `package.json`. Did you re-run `yarn` after updating it?

Reply:
yes i did, but yarn.locks don't get previous requirements removed afaik

Reply:
because -rc isn't shared with the stable release

Reply:
Sounds like a bug to me, but this is V1, so maybe it got fixed over time.

Reply:
I would be tempted to tweak this manually, but the file is explicitly annotated as "do not modify" so I guess we're good.